charts/scalardl-audit/templates/auditor/deployment.yaml

```diff
@@ -122,13 +162,27 @@ spec:
               command:
               - /usr/local/bin/grpc_health_probe
               - -addr=:40051
+              {{- if .Values.auditor.tls.enabled }}
+              - -tls
+              {{- if .Values.auditor.tls.caRootCertSecret }}
+              - -tls-ca-cert=/tls/certs/ca-root-cert.pem
+              {{- end }}
+              - -tls-server-name={{ .Values.auditor.tls.overrideAuthority }}
```

Comment:
Ditto.

Reply:
Thank you for your review!
I answered it on the ScalarDB Cluster PR side.
https://github.com/scalar-labs/helm-charts/pull/255#discussion_r1535123060